enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/DocumentVerificationRepository.java

```diff
@@ -93,4 +88,32 @@ public interface DocumentVerificationRepository extends JpaRepository<DocumentVe
             "AND d.usedForVerification = true")
     List<DocumentVerificationEntity> findAllWithPhoto(IdentityVerificationEntity identityVerification);
 
+    /**
+     * Return document verification IDs by the given identity verification ID and document statuses.
+     *
+     * @param identityVerificationIds identity verification IDs
+     * @param statuses document statuses
+     * @return identity verification IDs
+     */
+    @Query("SELECT d.id FROM DocumentVerificationEntity d " +
+            "WHERE d.identityVerification.id in :identityVerificationIds " +
+            "AND d.status in :statuses")
+    List<String> findDocumentVerificationsByIdentityVerificationIdsAndStatuses(Collection<String> identityVerificationIds, Collection<DocumentStatus> statuses);
```

Comment:
The method name is a bit confusing, because it uses naming which looks like Spring repository naming, however a `@Query` is specified. Using a shorter method name and not using `by` will resolve the issue.